cos-fleetshard-api/runtime/pom.xml

```diff
@@ -31,7 +31,7 @@
         <plugins>
             <plugin>
                 <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-bootstrap-maven-plugin</artifactId><executions>
+                <artifactId>quarkus-extension-maven-plugin</artifactId><executions>
```

Comment:
Is this related?

Reply:
yes, the bootstrap plugin has been deprecated in recent quarkus version in favor of the extension plugin